src/views/validated-designs/guide/index.tsx

```diff
@@ -25,9 +25,11 @@ import s from './detail-view.module.css'
 import type { HvdPage, HvdPageMenuItem } from '../types'
 import type { MDXRemoteSerializeResult } from 'next-mdx-remote'
 import type { OutlineLinkItem } from 'components/outline-nav/types'
+import { ProductSlug } from 'types/products'
 
 export interface ValidatedDesignsGuideProps {
 	title: string
+	productSlug: Exclude<ProductSlug, 'sentinel'>
```

Comment:
this couldn't be named `product` as we have a hook that when a react param named `product` is passed in it will render that products navigation bar components, but that is not something we want for HVD.